quirc/examples/esp-eye-qrcode/main/app_main.c

```diff
@@ -0,0 +1,85 @@
+#include <stdio.h>
+#include <stdint.h>
+#include <string.h>
+#include "esp_log.h"
+#include "esp_system.h"
+#include "freertos/FreeRTOS.h"
+#include "freertos/task.h"
+#include "app_peripherals.h"
+#include "quirc.h"
+
+static const char *TAG = "APP_CODE_SCANNER";
+
+static void decode_task()
+{
+    if(ESP_OK != app_camera_init()) {
+        vTaskDelete(NULL);
+        return;
+    }
+
+    camera_fb_t *fb = NULL;
+
+    // Initializing the quirc handle
+    struct quirc *q = quirc_new();
+    if (!q) {
+        ESP_LOGE(TAG,"Failed to allocate memory\n");
```

Comment:
Newline at the end of format strings is not necessary for ESP_LOG, it's added automatically.

Also, missing space after comma. (Have you installed the pre-commit hooks using `pre-commit install`? They should automatically format the source files.)